JsonLogic/Rules/LooseEqualsRule.cs

```diff
@@ -50,11 +51,14 @@ protected internal LooseEqualsRule(Rule a, Rule b)
 	}
 }
 
-internal class LooseEqualsRuleJsonConverter : JsonConverter<LooseEqualsRule>
+internal class LooseEqualsRuleJsonConverter : AotCompatibleJsonConverter<LooseEqualsRule>
 {
 	public override LooseEqualsRule? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 	{
-		var parameters = JsonSerializer.Deserialize<Rule[]>(ref reader, options);
+		var readerCopy = reader;
```

Comment:
Why did this need to change?

Reply:
I was just testing things. Wanted to see what was in there.  Trying to figure out the null issue.